README.md

```diff
@@ -9,229 +9,19 @@ https://tegel.scania.com/
-sdds-prefix. This is done in order to not have conflicting events and to make it clear to the user the specified event is something that is emitted
-from a tegel component.
-
-The events are named according to our naming convention: 'sdds' + event.
-For a click event this would result in the event being called `sddsClick`. To listen for these events in vanilla JS the event name
-should be passed to the `addEventListener` function as the first argument:
-
-```javascript
-document.addEventListener('sddsClick', (event) => {
-  // Do something with/based on the event.
-});
-```
-
-In React these events needs to be listened to by adding an event listener to the element that emits the event. This can be easily done by
-grabbing a reference to the element and adding the event listener inline:
-
-This solution is a workaround based on how React handles events, you can read more on this [here.](https://reactjs.org/docs/events.html)
-
-```jsx
-<sdds-textfield
-  ref={(element) =>
-    element.addEventListener('sddsClick', (event) => {
-      {
-        /* Do something with/based on the event. */
-      }
-    })
-  }
-></sdds-textfield>
-```
-
-In other JSX-enviroments (apart from React) these events can be listened to by prefixing them with an `on` directly on the component:
-
-```jsx
-<sdds-textfield
-  onSddsChange={(event) => {
-    /* Do something with/based on the event. */
-  }}
-></sdds-textfield>
-```
-
-What is attached to the event object is highlighted in our storybook. Under the docs tab of each component the events are outlined,
-inluding what data is passed with the event to the user.
+This package is currently in a **pre-beta** stage. We are now working hard towards a 1.0-beta release, but if you want to try out the package today you can! It's available via NPM and can be installed using the installation guide below.
 
-### Internal events
+## Getting started
 
-Some of the component are using event to communicate with its parent/child. These events are not recommended to use in any way since
-they might change without notice. Their payload might also be changed based on refactoring of components. These events are prefixed
-with 'internal'. This is to make it as clear as possible to a user that this is an internal event that the components are using,
-but the user should not interact with it. E.g. `internalSddsPropsChange`.
+Read about how to get started on our website: https://tegel.scania.com/development/getting-started-development
 
-## Browser support
+The website also includes information on browser support.
 
-See the browser support section on [the Tegel website](https://tegel.scania.com/development/getting-started-development/introduction#browser-support).
+## Contributing
 
-## Code conventions
+### Code conventions
 
 The code conventions used in (and enforced by) Tegel is documented [here](https://github.com/scania-digital-design-system/tegel/blob/main/.github/CODE_STYLE.md).
```

Comment:
Should we update the CODE_STYLE.md in a another PR? It contains a lot of mentions of sdds still.